test/tests/mqtrigger/kafka/test_kafka.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#test:disabled
+
+# Create a function and trigger it using Kafka
+# This test requires Kafka & MQ-Kafka component of Fission installed in the cluster
+set -euo pipefail
+set +x
+
+nodeenv="node-kafka"
+goenv="go-kafka"
+producerfunc="producer-func"
+consumerfunc="consumer-func"
+
+log() {
+    echo $1
+}
+export -f log
```

Comment:
I did not know you could do this, cool! However, why is it needed? It looks like the functions are only used within this file?

Reply:
I did this to make it work on mac (Like other scripts), but can be removed for now

Reply:
This was a fallback for using log on Mac, can be removed though.